source/_integrations/my.markdown

```diff
@@ -0,0 +1,24 @@
+---
+title: My Home Assistant
+description: Support for the my.home-assistant.io redirect service.
+ha_category:
+  - Other
+ha_release: 2021.3.0
+ha_domain: my
+ha_iot_class: Calculated
```

Comment:
The IoT class is not calculated, as this integration does not provide entities and does not dirrive states from other entities. In this case, it should be have nulled.

Reply:
https://github.com/home-assistant/home-assistant.io/pull/16458